src/js/api.js

```diff
@@ -71,8 +78,8 @@ export default class API {
    * POST /articles
    * @return {Promise}
    */
-  createArticle(fields) {
-    return this._wrap(this.client({method: "POST", path: "/articles", entity: fields}));
+  createArticle(params={}) {
+    return this._wrap(this.client({method: "POST", path: "/articles", entity: params}));
```

Comment:
``params`` may not be the most relevant name here. I would prefer fields, record, entity...